regression/contracts/loop_assigns-slice-assignable-scalar/test.desc

```diff
@@ -0,0 +1,23 @@
+CORE
+main.c
+--apply-loop-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[main.\d+\] .* Check loop invariant before entry: SUCCESS$
+^\[main.assigns.\d+\] .* Check that i is assignable: SUCCESS$
+^\[main.assigns.\d+\] .* Check that b->data\[\(.*\)0\] is assignable: SUCCESS$
+^\[main.assigns.\d+\] .* Check that b->data\[\(.*\)i\] is assignable: FAILURE$
+^\[main.\d+\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assigns.\d+\] .* Check that __CPROVER_assignable\(\(.*\)&b->data\[\(.*\)0\], 1ul, FALSE\) is valid: SUCCESS$
```

Comment:
This is failing on Windows, where `1ul` is `1ull` instead. I believe `1ul?l?` is the right pattern to cater to various platforms.

Reply:
Thank you!